From 218b99f1e4ee7cd9f3df0a5624a35bd2926c3a96 Mon Sep 17 00:00:00 2001
From: Kalyan Kondapally <kalyan.kondapally@intel.com>
Date: Wed, 20 Aug 2014 15:28:27 -0700
Subject: [PATCH] [WebRTC] Remove Dependencies on X11.

Temporary patch to remove any x11 dependencies in WebRTC.
This should be removed once the following CL is merged:
https://webrtc-codereview.appspot.com/13999004/
---
 base/base.gyp |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/base/base.gyp b/base/base.gyp
index 0e5845e..cbabd00 100644
--- a/base/base.gyp
+++ b/base/base.gyp
@@ -574,10 +574,6 @@
             'libraries': [
               '-ldl',
               '-lrt',
-              '-lXext',
-              '-lX11',
-              '-lXcomposite',
-              '-lXrender',
             ],
           },
         }, {
-- 
1.7.9.5

